### PR TITLE
Make font settings react to pref changes

### PR DESCRIPTION
**Pull request by marcelgerber**
This should go with, or before, #11130.

### CODE REVIEW

File: src/view/ViewCommandHandlers.js [hunk 1]
```diff
@@ -49,6 +49,10 @@ define(function (require, exports, module) {
 
     var prefs = PreferencesManager.getExtensionPrefs("fonts");
 
+    // These variables contain the preference values. They are used to no-op in case nothing changed
+    // and  updated *after* the call to setFontSize/setFontFamily.
+    var fontSizeValue, fontFamilyValue;
```
Comment (MiguelCastillo) [inline src/view/ViewCommandHandlers.js]: Let's add proper jsdoc to these two variables and mark them as private.

**Comment by MiguelCastillo** [comment]
@MarcelGerber changes look good.  I am not even sure why I didn't add this with the initial PR, so thanks :)

**Comment by nethip** [comment]
@MarcelGerber thanks for this change! I just have a s comment. Can we change the variable name to something like currFontSizeValue.  I guess it would improve the readability of the code. Otherwise LGTM.

**Comment by MarcelGerber** [comment]
Changes pushed.

**Comment by MiguelCastillo** [comment]
looks good, works well.  merging in.

**Pull request merged**